libs/deal-ticket/src/utils/validate-size.ts

```diff
@@ -0,0 +1,12 @@
+export const ERROR_SIZE_DECIMAL = 'step';
+
+export const validateSize = (step: number) => {
+  const [_, stepDecimals = ''] = String(step).split('.');
+  return (value: string) => {
+    const [_, valueDecimals = ''] = value.split('.');
```

Comment:
follow up (non-blocking): This will not work depending on the localisation of the platform (ie. if commas are used instead of peroids. Will need to add logic for this.